### PR TITLE
Refresh /desktop/developers page 

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**Note: Waiting on 22.04 screenshots**

- Updates  /desktop/developers page for the 22.04 refresh, see [copydoc](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit)

## QA

- Go to https://ubuntu-com-11493.demos.haus/desktop/developers and compare against [copydoc](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4960

### CODE REVIEW

**Comment by local-optimum** [comment]
Feedback (wrong copydoc btw, right one is linked here https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit)
- Both laptop images needs updating, particularly the second one which is very out of date.
- The graph on what developers prefer should be next to the 'in good company' text section
- Would be good to have the Landscape icon in the landscape section
- Would be good to add the Rust logo to the 'tools' image

File: templates/desktop/developers.html [hunk 1]
```diff
@@ -34,97 +34,80 @@ <h2>Why use Ubuntu for your development workstation?</h2>
...
-        <li class="p-list__item is-ticked">Active Directory integration with the Ubuntu installer</li>
-        <li class="p-list__item is-ticked">Private home directory</li>
-        <li class="p-list__item is-ticked">Recovery key option for encrypted installations</li>
-      </ul>
-      <p>
-        <a href="{{ releases.latest.release_notes_url }}">
-          Read the full release notes&nbsp;&rsaquo;
-        </a>
-      </p>
+      <h2>With Ubuntu, you're in good company</h2>
```
Comment (mtruj013) [inline templates/desktop/developers.html:65]: Is this supposed to be on the same section as the image above?

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Looks good @petesfrench! Just a few comments, the majority of which are about wrapping dashes so nothing major. Also, TIL that "artifacts" is spelled "artefacts" in the UK 😅

File: templates/desktop/developers.html [hunk 2]
```diff
@@ -315,6 +298,30 @@ <h2>Support tailored to developers' needs</h2>
     </div>
   </section>
 
+  <section class="p-strip">
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/desktop/developers.html:245]: You are correct but the combination of tutorials isn't currently possible so I just did it manually for now

File: templates/desktop/developers.html [hunk 3]
```diff
@@ -8,22 +8,21 @@
   <section class="u-image-position p-strip--suru-topped">
     <div class="row u-equal-height">
       <div class="col-6">
-        <h1>Ubuntu Desktop for&nbsp;developers</h1>
-        <p>Whether you're a mobile app developer, an engineering manager, a music or video editor or a financial analyst with large-scale models to run &mdash; in fact, anyone in need of a powerful machine for your work &mdash; Ubuntu is  your ideal workstation.</p>
+        <h1>Ubuntu Desktop for&nbsp;Linux developers</h1>
+        <p>Whether you're an app or web developer, engineering manager, data scientist, or AI/ML researcher&mdash;Ubuntu is  your ideal workstation.</p>
```
Comment (anthonydillon) [inline templates/desktop/developers.html:12]:
```suggestion
        <p>Whether you're an app or web developer, engineering manager, data scientist, or AI/ML researcher &mdash; Ubuntu is your ideal workstation.</p>
```

File: templates/desktop/developers.html [hunk 4]
```diff
@@ -8,22 +8,21 @@
...
-            height="300",
-            width="500",
+            width="1000",
+            height="578",
             hi_def=True,
-            attrs={"class": "col-6 u-vertically-center u-hide--medium u-hide--small u-align--center"},
             loading="auto",
-          ) | safe
-        }}
+            attrs={"class": "col-6 u-vertically-center u-hide--medium u-hide--small u-align--center"},
```
Comment (anthonydillon) [inline templates/desktop/developers.html:23]: These classes are on the parent not the image

File: templates/desktop/developers.html [hunk 5]
```diff
@@ -34,109 +33,90 @@ <h2>Why use Ubuntu for your development workstation?</h2>
...
         <li class="p-list__item is-ticked">Offers the broadest and best development tools and libraries</li>
-        <li class="p-list__item is-ticked">Has all the most popular productivity apps such as Slack, Skype, Telegram and Discord, available in the snap store</li>
         <li class="p-list__item is-ticked">Features official snaps for Visual Studio Code and the JetBrains suite of IDEs</li>
-        <li class="p-list__item is-ticked">Makes for hassle free gaming and AI development with NVIDIA GPUs supported out the box</li>
-        <li class="p-list__item is-ticked">Certified AI and developer laptops and workstations are available from Dell, Lenovo and HP – get a seamless, pre-installed Ubuntu experience</li>
-        <li class="p-list__item is-ticked">The lightweight OS runs natively or in a VM, ideal for any resource-intensive environment, from data mining to large-scale financial modelling</li>
+        <li class="p-list__item is-ticked">Has all the most popular productivity apps such as Slack, Microsoft Teams, Telegram, and Discord, available in the Snap Store</li>
```
Comment (anthonydillon) [inline templates/desktop/developers.html:37]: Missing Zoom

File: templates/desktop/developers.html [hunk 6]
```diff
@@ -34,109 +33,90 @@ <h2>Why use Ubuntu for your development workstation?</h2>
...
+            <li>Compact and modern on-screen display notifications</li>
+            <li>New screenshot/screen recording interface</li>
+            <li>New power profile selection</li>
+            <li>Improved multi-monitor support</li>
+          </ul>
+        </li>
+        <li class="p-list__item is-ticked">GNOME desktop performance improvements
+          <ul>
+            <li>Up to 2X improvement in frame rates with Intel and Raspberry Pi graphics drivers</li>
+            <li>Significant improvements for AMD and NVidia</li>
```
Comment (anthonydillon) [inline templates/desktop/developers.html:92]:
```suggestion
            <li>Significant improvements for AMD and NVIDIA</li>
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few comments in the code

File: templates/desktop/developers.html [hunk 7]
```diff
@@ -288,17 +215,17 @@ <h2>By developers, for developers</h2>
         <h2>Deployment made easy</h2>
```
Comment (anthonydillon) [inline templates/desktop/developers.html:215]: This image could probably be hidden on small screens.

**Review by anthonydillon — APPROVED** [review]
Thanks LGTM :+1:

**Pull request merged**